### PR TITLE
Update paging sample to alpha13

### PR DESCRIPTION
**Pull request by dlam**
- Cleanup with viewbinding
- Bumping to alpha13 with getRefreshKey implementations

Resolves: #942
Resolves: #571

### CODE REVIEW

File: PagingWithNetworkSample/app/src/test/java/com/android/example/paging/pagingwithnetwork/reddit/repository/SubredditPagingSourceTest.kt [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        assertEquals(
+            expected = Page(
+                data = listOf(fakePosts[0]),
+                prevKey = fakePosts[0].name,
+                nextKey = fakePosts[0].name
+            ),
+            actual = pagingSource.load(
+                Refresh(
+                    key = null,
+                    loadSize = 1,
```
Comment (ChrisCraik) [inline PagingWithNetworkSample/app/src/test/java/com/android/example/paging/pagingwithnetwork/reddit/repository/SubredditPagingSourceTest.kt:39]: suggest using loadSize = 2, so the prevKey and nextKey behavior are better covered

File: PagingWithNetworkSample/app/src/test/java/com/android/example/paging/pagingwithnetwork/reddit/repository/SubredditPagingSourceTest.kt [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        fakePosts.forEach { post -> addPost(post) }
+    }
+
+    @Test
+    fun itemKeyedSubredditPagingSource() = runBlockingTest {
+        val pagingSource = ItemKeyedSubredditPagingSource(fakeApi, DEFAULT_SUBREDDIT)
+        assertEquals(
+            expected = Page(
+                data = listOf(fakePosts[0]),
+                prevKey = fakePosts[0].name,
```
Comment (ChrisCraik) [inline PagingWithNetworkSample/app/src/test/java/com/android/example/paging/pagingwithnetwork/reddit/repository/SubredditPagingSourceTest.kt:33]: if key is null, why isn't prevKey == null?

do we need to query remote to know there aren't any items before?

**Review by ChrisCraik — APPROVED** [review]

File: PagingWithNetworkSample/app/src/test/java/com/android/example/paging/pagingwithnetwork/reddit/repository/SubredditPagingSourceTest.kt (hunk 2 above)
Comment (dlam) [reply PagingWithNetworkSample/app/src/test/java/com/android/example/paging/pagingwithnetwork/reddit/repository/SubredditPagingSourceTest.kt:33]: It's due to how our `ItemKeyedPagingSource` sample is implemented. It essentially loads until its empty.

```
prevKey = items.firstOrNull()?.name,
nextKey = items.lastOrNull()?.name
```

File: PagingWithNetworkSample/app/src/test/java/com/android/example/paging/pagingwithnetwork/reddit/repository/SubredditPagingSourceTest.kt (hunk 1 above)
Comment (dlam) [reply PagingWithNetworkSample/app/src/test/java/com/android/example/paging/pagingwithnetwork/reddit/repository/SubredditPagingSourceTest.kt:39]: Done

**Pull request merged**